app/workers/articles/detect_animated_images_worker.rb

```diff
@@ -0,0 +1,17 @@
+module Articles
+  class DetectAnimatedImagesWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 5, lock: :until_executing
```

Comment:
My vote would be for medium_priority

Reply:
Same. I usually reserve high-priority queues for truly urgent operations — this is domain-specific, but in our case it might be things like cache busting, Slack notifications. Anything that is expected to happen in near-real-time.

Most of the time, the queue is empty and it doesn't matter what the priorities are, but when it's not (for example, we broke something in our Sidekiq config and no jobs were being processed for a while), we need the most urgent things to run first. The best way I've heard it described is "if everything's high-priority, then nothing's high-priority".

IME medium is a great default, so that if something you're working on can wait a few minutes (reporting queries, for example) you can drop it to low, or if something is truly urgent you can bump it up to high.